src/Domain/Fixture/FixtureCustomizer.php

```diff
@@ -117,4 +118,33 @@ public function removePerzParagraphsTests(FixtureOptions $options): void {
 
   }
 
+  /**
+   * Modifies DrupalKernel.php to suppress Drupal 9 deprecation warnings.
+   */
+  public function modifyDrupalKernel(FixtureOptions $options): void {
+    if (!$options->coreVersionParsedMatches('^9')) {
+      return;
+    }
+
+    if (!$this->fixturePathHandler->exists('docroot/core/lib/Drupal/Core/DrupalKernel.php')) {
```

Comment:
You should consider making this string a variable, since you use it multiple times--for readability and maintainability. For example:

```suggestion
$drupal_kernel_path = 'docroot/core/lib/Drupal/Core/DrupalKernel.php';

    if (!$this->fixturePathHandler->exists($drupal_kernel_path)) {
```

...and obviously use the variable below, too.

Reply:
Done